frontend/app/assets/javascripts/spree/frontend/multi_currency.js.coffee

```diff
@@ -0,0 +1,9 @@
+$ ->
```

Comment:
@PSzyszkaSpark we need this file in plain JS, not coffee, we've dropped coffee support in 3.7